### PR TITLE
otl2html.py should generate a proper HTML5 pages.

### LINKED ISSUE #115: otl2html should recognize URLs
It would be lovely if `otl2html` recognized URLs in the outline and marked them as `<a>` element.

### PR DESCRIPTION
**Pull request by mcepl**
Generally improve generated HTML page. It is now proper HTML5
page with set encoding, but still non-validating (see
https://validator.w3.org/).

Fix #115

### CODE REVIEW

File: vimoutliner/scripts/otl2html.py [hunk 1]
```diff
@@ -1076,6 +1079,7 @@ def printFooter():
 
 def main():
     global showTitle
+    locale.setlocale(locale.LC_ALL, '')
```
Comment (ressu) [inline vimoutliner/scripts/otl2html.py]: Why does it reset LC_ALL?

Comment (mcepl) [inline vimoutliner/scripts/otl2html.py]: Please, read https://docs.python.org/2.7/library/locale.html … it does something else than you seem to think it does.

Comment (ressu) [inline vimoutliner/scripts/otl2html.py]: Wow, you are right. That's one curious way of getting defaults.

**Comment by mcepl** [comment]
Nobody commented on this for two years, so I guess silence is an approval.

**Pull request merged**